ignite/pkg/protoanalysis/protoutil/parser.go

```diff
@@ -0,0 +1,36 @@
+package protoutil
+
+import (
+	"io"
+	"os"
+	"strings"
+
+	"github.com/emicklei/proto"
+	"github.com/emicklei/proto-contrib/pkg/protofmt"
+)
+
+// ParseProtoFile opens the file denoted by path and parses it
+// into a proto file.
+func ParseProtoPath(path string) (pf *proto.Proto, err error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	return proto.NewParser(f).Parse()
+}
+
+// PraseProtoFile parses the given file.
+func ParseProtoFile(r io.Reader) (*proto.Proto, error) {
+	return proto.NewParser(r).Parse()
+}
+
+// Printer() formats the proto file using proto-contrib/pkg/protofmt.
```

Comment:
```suggestion
// Printer formats the proto file using proto-contrib/pkg/protofmt.
```

I also think renaming to `Print` is a better name.